util/chplenv/third-party-pkgs

```diff
@@ -65,7 +65,8 @@ for l in chplenv.split('\n'):
         var=var.strip()
         val=val.strip()
         # Ignore platform variables
-        if not var.endswith('_PLATFORM'):
+        # Ignore CHPL_MEM (use CHPL_JEMALLOC)
+        if not (var.endswith('_PLATFORM') or var == 'CHPL_MEM'):
```

Comment:
I added `or var == 'CHPL_MEM' ` so if CHPL_MEM=jemalloc, we don't build third-party/jemalloc